frontend/lib/main.dart

```diff
@@ -52,3 +46,19 @@ Future<void> main() async {
     run();
   }
 }
+
+Future<String?> loadLocaleOverride(AppLocale locale) async {
+  final path = '${buildConfig.localeOverridePath}/override_${locale.languageCode}.json';
+  try {
+    return await rootBundle.loadString(path);
+  } catch (e) {
+    return null;
```

Comment:
:question: Can this happen in production? Is it fine to just ignore this exception?

Reply:
Well, we could actually check the exception message, and only ignore if it's "asset doesn't exist". Maybe it would make sense

Reply:
:question: I think we should definitely handle this, thanks. However, shouldn't we display at least a warning message to the console in case this is not missing on purpose?

Reply:
Added message check and log message, thanks for the ideas :)